### PR TITLE
Presets: minor fixes

### PR DESCRIPTION
**Pull request by limonspb**
Four small changes/fixes:
1. Preselect not just TUNE but also RC_LINK, RATES and RC_SMOOTHING as probably the four most popular things.
2. Allow #$ INCLUDE of the files with upper case.
3. On the preset source dialog when making active a source it will save it's changes it automatically (if name, link were changed)
4. Small style change for the preset filter fields when something is selected there:
image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

File: src/tabs/presets/presets.css
Comment (chmelevskij) [inline src/tabs/presets/presets.css:130]: 😆 does it have to  be `!important` usually this will lead to more headscrathing than solved problems

File: src/tabs/presets/presets.js
Comment (chmelevskij) [inline src/tabs/presets/presets.js:379]: Just cast this to boolean instead of yoda comparisons 😉 
```suggestion
    const isSomethingSelected = !!selectedOptions.length;
```

File: src/tabs/presets/presets.css
Comment (chmelevskij) [inline src/tabs/presets/presets.css:128]: any reason why presets is the only file to use snake_case? Would be nice to agree on something a bit more consistent since we have literally every casing option possible when it comes to css 😄

**Review by chmelevskij — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

File: src/tabs/presets/presets.css
Comment (limonspb) [reply src/tabs/presets/presets.css:130]: This is a sort of hacking for multiSelect jquery component styling. The original styles and html are defined within the component.

File: src/tabs/presets/presets.css
Comment (limonspb) [reply src/tabs/presets/presets.css:128]: Not the only file for sure, for example main.css itself :)
But I agree we need to define CSS/HTML coding style 100%

**Pull request merged**
